connect_game.py

```diff
@@ -93,7 +93,7 @@ def update(self):
         Checks the game state, dispatching events as needed.
         """
         if self.game_data.game_board.tie_move():
-            bus.emit("game:over", GameOver(was_tie=True))
+            bus.emit("game:over", self.renderer, GameOver(was_tie=True)) #There was a syntax error in this line
```

Comment:
please remove the one line comment.

Reply:
should i remove all the comments in it or only that one in line96
i have comment even while fixing the issue#21

Reply:
yes, remove those too.

Reply:
ok 2 minutes